scripts/routes.js

```diff
@@ -57,8 +66,9 @@ export default function () {
 
 export function handleSummary (data) {
   // removing default metrics
-  deleteRoutesMetrics(data)
```

Comment:
Just a note, attempted to put the handleSummary internal logic to a function that I can reuse everywhere, but that causes differences when running with `k6 run` and `./k6 run` for some reason, so decided to leave the duplication.